### PR TITLE
Service creation using network plugin filters.

### PR DESCRIPTION
**Pull request by anusha-ragunathan**
Signed-off-by: Anusha Ragunathan <anusha.ragunathan@docker.com>

### CODE REVIEW

File: docs/extend/plugins_services.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+<!-- This file is maintained within the docker/docker Github
+     repository at https://github.com/docker/docker/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# Service creation using plugin filters.
```
Comment (mavenugo) [inline docs/extend/plugins_services.md]: If this is the only existing documentation to introduce the plugin-v2 concept being used along-side services, then the title should not be narrowly focussing on `filters`.
IMO, it should be more generic such as `Volume and Network plugins for Docker services` (also the existing title is a bit misleading ... the plugins themselves doesnt assist the service creation. The plugins are merely drivers that replace the default drivers).

File: docs/extend/index.md [hunk 2]
```diff
@@ -118,61 +118,6 @@ remove it, use the `docker plugin remove` command. For other available
 commands and options, see the
 [command line reference](../reference/commandline/index.md).
 
-## Service creation using plugins
```
Comment (mavenugo) [inline docs/extend/index.md]: I see that this content is moved to another file. How are they linked ?

Comment (anusha-ragunathan) [inline docs/extend/index.md]: Earlier service creation using plugin filters was part of the generic index file. Services and plugins deserve a separate file, hence the move.

File: docs/extend/plugins_services.md (hunk 1 above)
Comment (anusha-ragunathan) [inline docs/extend/plugins_services.md]: Updated.

**Review by vieux — APPROVED** [review]
LGTM

**Pull request merged**